apps/system/test/marionette/statusbar_icon_color_test.js

```diff
@@ -77,9 +77,10 @@ marionette('Statusbar colors', function() {
   test('statusbar icons keep color after add homescreen', function() {
     waitVisible();
     helper.unlockScreen(client);
+    waitForDarkColor();
     var url = server.url('sample.html');
     bookmark.openAndSave(url);
-    waitForLightColor();
+    waitForDarkColor();
```

Comment:
Yes! I thought that change was done on purpose! Anyway, it seems that these tests are not working properly now...
I'll leave them as they were, and make the bug dependent on 1168915. Thanks!